gwvolman/lib/dataone/publish.py

```diff
@@ -355,9 +400,9 @@ def _extract_user_info(self, jwt_token):
         :return: The ORCID ID
         :rtype: str, None if failure
         """
-        jwt_token = jwt.decode(jwt_token, verify=False)
-        user_id = jwt_token.get('userId')
-        name = jwt_token.get('fullName')
+        jwt_token = jwt.JWT().decode(self.dataone_auth_token, do_verify=False)
```

Comment:
I get an error here `AttributeError: module 'jwt' has no attribute 'JWT'`. Wondering if this should still just be `jwt.decode()`

Reply:
Fixed in e57f4dc (now supports multiple versions of jwt)